src/components/dashboard/subscriptions/SubscriptionContainer.js

```diff
@@ -13,8 +12,16 @@ class SubscriptionContainer extends Component {
 		super(props);
 		this.changeBind = this.change.bind(this);
 		this.deleteBind = this.delete.bind(this);
-		this.update = ActionUtil.wrapPagedActionWithId(this.props.user.id,this.props.dispatch, getSubscriptionsByUser);
-	} 
+		this.updateBind = this.update.bind(this);
+	}
+
+	update(page) {
```

Comment:
@yuderekyu as you suspected, we were running into issues with this page trying to load before we had loaded this.props.user. This seems like an ugly way of waiting, but it prevents us from trying to load if we don't have a userId. 

It still feels kinda dirty, so if there's a better idea floating around, go ahead. It's also an option to get the userID from authenticated requests now via the `X-UserID` parameter inside any request that's been authenticated. 

The only weirdness with that is trying to figure out a nice way to support both userID by token and userID by url param from the api.